components/builder-api/src/depot/server.rs

```diff
@@ -965,7 +973,17 @@ fn upload_package(req: &mut Request) -> IronResult<Response> {
                         &filename, e
                     ),
                 }
-
+                match set_memcache_package(
+                    &mut client,
+                    ident.get_origin().to_string(),
```

Comment:
Just pass in the full ident struct as a reference?

Reply:
oh that's a good idea since PackageIdent versions and revisions are also optional!